### DISCUSSION TITLE
negative frequency using the grid huge

### DISCUSSION THREAD

**Discussion by DrKNiu** [nwchemgit/nwchem#1024] [Q&A]
I want to optimize the structure to an energy minimum with grid huge option and It still has several negative frequencies. May I know how to avoid the negative frequency please? here is some other solutions for metal. https://nwchemgit.github.io/Special_AWCforum/st/id2535/Trouble_with_negative_frequencie....HTML
The input file is as follows.
title "pdi  b3lyp 3-21g(d,) geometry optimization"  
memory 32 gb
scratch_dir ./scratch
permanent_dir ./perm
#restart
geometry nocenter noautosym noautoz
 C   -1.03654482    -1.15018255    -3.42618529
 C    0.05395608    -0.54116394    -1.31342301
 C   -1.19548108    -0.06651683    -0.82172140
 C   -2.30682593    -0.15457013    -1.65189044
 C   -2.23990850    -0.68826264    -2.94067954
 C    1.24254862    -0.48956868    -0.53258066
 C   -1.25676428     0.49362339     0.53509842
 C   -0.06527099     0.54301479     1.31270722
 C    1.18190247     0.06685658     0.81675033
 C   -1.33174113     1.56341616     3.14942754
 C   -2.47948328     1.51121343     2.38847012
 C   -2.43115200     0.98035581     1.09806156
 H   -3.11650817    -0.74852546    -3.57276947
 H   -3.40315106     1.88596186     2.81043056
 C    1.32949449    -1.56275885    -3.14877686
 C    2.46570914    -1.50458794    -2.37995458
 C    2.41827271    -0.97133417    -1.08391429
 H    3.39221916    -1.86993916    -2.77921570
 C    2.29515232     0.14948964     1.63541249
 C    1.04287739     1.14662763     3.42173226
 C    2.22608469     0.68589162     2.92921110
 H    3.10675670     0.73730481     3.54045318
 C   -0.11850981     1.08353413     2.62544726
 C    0.99497409     1.71056583     4.79740958
 C   -1.41523386     2.13158704     4.51512410
 C    0.11844563    -1.08374135    -2.62536310
 C   -0.99499962    -1.71060356    -4.79739650
 C    1.41521585    -2.13159755    -4.51512116
 O    2.43608940    -2.53997960    -4.96655929
 O   -1.96459032    -1.77112806    -5.48225961
 N    0.23713215    -2.16712848    -5.25399836
 C    0.29815569    -2.71606953    -6.59230258
 C    0.08072135    -4.07638057    -6.76816711
 C    0.57188410    -1.86230567    -7.65284063
 C    0.14344012    -4.58492105    -8.05828546
 C    0.62710584    -2.40521781    -8.92928071
 C    0.41457099    -3.75605449    -9.13075245
 H   -0.02101279    -5.63498559    -8.22197076
 H    0.83749910    -1.76540472    -9.76766469
 H    0.46037708    -4.16384013   -10.12501667
 C   -0.21613266    -4.98655156    -5.60053514
 H   -0.35466169    -6.00483637    -5.94285670
 C    0.80237298    -0.38537327    -7.43728987
 H   -0.07140584     0.08767103    -6.99985889
 N   -0.23714632     2.16714408     5.25399187
 C   -0.29816934     2.71602634     6.59232047
 C   -0.57192824     1.86223382     7.65282708
 C   -0.08067476     4.07632144     6.76823809
 C   -0.62706007     2.40508895     8.92929543
 C   -0.14329768     4.58480364     8.05838472
 C   -0.41441273     3.75589801     9.13082618
 H   -0.83747004     1.76525293     9.76765778
 H    0.02121474     5.63485244     8.22211284
 H   -0.46013714     4.16363545    10.12511345
 C   -0.80249838     0.38531992     7.43723256
 H   -1.64762339     0.21003608     6.77893636
 C    0.21618613     4.98652537     5.60063639
 H   -0.59823250     4.98405245     4.88290688
 O    1.96457847     1.77113646     5.48224885
 O   -2.43609087     2.54003963     4.96653430
 H   -1.00955312    -0.10618233     8.38019379
 H    0.07150181    -0.08787065     7.00038538
 H    1.12272823     4.68361915     5.08636618
 H    0.35402020     6.00492027     5.94290971
 H    1.64789599    -0.21003373    -6.77951190
 H    1.00875167     0.10622447    -8.38034930
 H   -1.12228557    -4.68322639    -5.08583128
 H    0.59860679    -4.98458360    -4.88316071
 H   -3.33171989     0.94845070     0.52556362
 H    3.24895477    -0.19802131     1.29507303
 H    3.33171373    -0.94838413    -0.52558656
 H   -3.24895591     0.19808666    -1.29510230

end

 basis  
   * library 3-21g*
 end  
 dft
  grid huge
  xc b3lyp
 end
scf
  maxiter 200
  semidirect memsize 400000000 filesize 0
end
 task dft optimize 
 task dft freq numerical 

The output for the frequency part is shown below.
 ----------------------------------------------------------------------------
 Normal Eigenvalue ||         Derivative Dipole Moments (debye/angs)
  Mode   [cm**-1]  ||      [d/dqX]             [d/dqY]           [d/dqZ]
 ------ ---------- || ------------------ ------------------ -----------------
    1      -62.265 ||       0.291              -0.082             0.025
    2      -52.276 ||       0.045               0.003            -0.003
    3      -48.760 ||      -0.030               0.029            -0.005
    4      -29.153 ||       0.076              -0.021            -0.015
    5      -22.550 ||       0.050               0.018            -0.047
    6      -18.472 ||       0.066               0.006             0.007
    7        8.414 ||       0.082              -0.001             0.007
    8       14.080 ||      -0.050               0.028            -0.010
    9       30.825 ||      -0.047               0.009            -0.002
   10       42.290 ||       0.029               0.040             0.084
   11       47.319 ||      -0.037               0.007            -0.057
   12       51.165 ||      -0.022              -0.091             0.008
   13       55.284 ||       0.068               0.226            -0.022
   14       66.026 ||       0.024               0.017             0.006
   15       76.553 ||      -0.004              -0.006             0.003
   16       92.047 ||       0.184               0.025             0.007
   17       95.885 ||       0.026               0.031            -0.040

**Comment by edoapra** [comment]
Please post this discussion to the users' forum.
https://groups.google.com/g/nwchem-forum
